src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -226,11 +226,32 @@ public static IEnumerable<object[]> Abs_Advanced_TestData()
                 foreach (double invalidImaginary in s_invalidDoubleValues)
                 {
                     yield return new object[] { RandomPositiveDouble(), invalidImaginary, Math.Abs(invalidImaginary) }; // Invalid imaginary
-                    yield return new object[] { invalidReal, invalidImaginary, (double.IsInfinity(invalidReal) || double.IsInfinity(invalidImaginary)) ? double.PositiveInfinity : double.NaN }; // Invalid real, invalid imaginary
+                    yield return new object[] { invalidReal, invalidImaginary, (double.IsNaN(invalidReal) || double.IsNaN(invalidImaginary)) ? double.NaN : double.PositiveInfinity }; // Invalid real, invalid imaginary
```

Comment:
You probably want `[SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework | TargetFrameworkMonikers.NetcoreUwp)]`

And the net core specific test `[SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp)]`

There isn't that much of this divergence, and as and when we port to Netfx/desktop, we remove it. The benefit is the single set of tests that help us detect inadvertent behavioral divergence.